doc/design/design_cluster_job_runner.md

```diff
@@ -0,0 +1,126 @@
+# SQLFlow Cluster Job Runner
```

Comment:
The PR title does not mean the same thing as the markdown title

Reply:
Thanks and updated.